application/conns.go

```diff
@@ -110,6 +110,11 @@ func (cm *connManager) handleNewConn(regManager *cj.RegistrationManager, clientC
 	}
 	fd.Close()
 
+	cm.handleNewTCPConn(regManager, clientConn, originalDstIP)
+}
+
+func (cm *connManager) handleNewTCPConn(regManager *cj.RegistrationManager, clientConn net.Conn, originalDstIP net.IP) {
+	// logger := sharedLogger
```

Comment:
You should be able to uncomment this logger line to use the `sharedLogger` which is a global value (within the `main` package). You can then remove line 126.

Reply:
Would removing line 126 get rid of the clientIP logging? I thought it would and figured we still wanted to keep that log, so I removed line 119 to avoid the warning that the line 119 value of logger was never used since it was overwritten in line 126.